django/db/migrations/autodetector.py

```diff
@@ -180,6 +180,7 @@ def _detect_changes(self, convert_apps=None, graph=None):
         self.generate_removed_fields()
         self.generate_added_fields()
         self.generate_altered_fields()
+        self.generate_altered_indexes()
```

Comment:
you probably want to handle this afer the `generate_altered_index_together()` method.